be/src/runtime/snapshot_loader.cpp

```diff
@@ -168,7 +292,10 @@ Status SnapshotLoader::upload(const std::map<std::string, std::string>& src_to_d
         for (auto& local_file : local_files) {
             RETURN_IF_ERROR(_report_every(10, &report_counter, finished_num, total_num,
                                           TTaskType::type::UPLOAD));
-
+            if (_end_with(local_file, ".hdr")) {
+                RETURN_IF_ERROR(upload_remote_file(*_remote_fs, tablet_id, src_path, dest_path,
```

Comment:
No need to find .hdr file in this loop because it must exists, put this step after this loop?